### PR TITLE
Add Western asia databundle

### PR DESCRIPTION
**Pull request by Emre-Yorat89**
# Closes # (if applicable).

## Checklist

- [x] I consent to the release of this PR's code under the AGPLv3 license and non-code contributions under CC0-1.0 and CC-BY-4.0.
- [ ] I tested my contribution locally and it seems to work fine.
- [ ] Code and workflow changes are sufficiently documented.
- [ ] Newly introduced dependencies are added to `envs/environment.yaml` and `doc/requirements.txt`.
- [ ] Changes in configuration options are added in all of `config.default.yaml` and `config.tutorial.yaml`.
- [ ] Add a test config or line additions to `test/` (note tests are changing the config.tutorial.yaml)
- [ ] Changes in configuration options are also documented in `doc/configtables/*.csv` and line references are adjusted in `doc/configuration.rst` and `doc/tutorial.rst`.
- [ ] A note for the release notes `doc/release_notes.rst` is amended in the format of previous release notes, including reference to the requested PR.

### CODE REVIEW

File: configs/bundle_config.yaml
Comment (davide-f) [inline configs/bundle_config.yaml:373]: Amazing @Emre-Yorat89 :D
This seems to be ok, the only comment I'd recommend is to add a comment on top of the bundle similarly to the others.
You may use Asia or Africa as templates.

File: configs/bundle_config.yaml
Comment (davide-f) [inline configs/bundle_config.yaml]: Please, restore bundle_name

**Review by davide-f — COMMENTED** [review]
Amazing @Emre-Yorat89.
This PR is almost ready to go, few comments only :)

**Comment by ekatef** [comment]
Hey @Emre-Yorat89! As discussed, troubles with databundle you experience may be linked with the issue recently fixed by #844. In case updating the repo doesn't help, could you please add an issue on the problem you observe? It would be very helpful to increase stability of the model

File: configs/bundle_config.yaml
Comment (ekatef) [inline configs/bundle_config.yaml]: Could you please rename file to "cutout-2013-era5.nc" to make it consistent with other cutouts names?

**Comment by ekatef** [comment]
Hello @Emre-Yorat89! Thanks for looking into the issue with databundle download. It is in fact connected with "too many request" response from google server side (added a comment to #853 on that). That is quite annoying, but the good news is that the issue is not in any way caused by your PR. Feel free please to finalise it :)

You may absolutely replace the current link with an updated one and fix two comments above (one of which I have just corrected, sorry for the typo before!). Then, ready to merge, I think.

**Comment by ekatef** [comment]
Hey @Emre-Yorat89! Thanks a lot for updating the link and introducing the fixes.
Great work and ready to merge 🚀 

The only point left is your consent to contribute under AGPL license which is used in the project. Would you mind to tick the first box in the check list to agree?

**Review by ekatef — APPROVED** [review]

**Review by davide-f — COMMENTED** [review]
Katia advised me that she is working on finalizing the PR.
I see that the git merge has been delicate.

To try to fix it, I'd recommend to:
- copy this branch to avoid losing the changes
- hard reset the branch before the merge that may have created issues. I believe a good commit may be:73d72a7783ce0985588df183439e65fab82bc20c . So the code is: git reset --hard 73d72a7783ce0985588df183439e65fab82bc20c and then git push --force

This should restore the version before the merge went wrong

**Comment by ekatef** [comment]
Sorry @Emre-Yorat89 for taking a while to prepare PR for merge. It has been a need to add a release note and resolve merge conflicts.

@davide-f thanks a lot for git spells! It has been really helpful. My feeling is that now PR is ready. IC on mac fails due to an error which looks suspiciously similar to troubles with google drive... I'd wait until IC-mac work and then merge. Would you agree?

**Comment by davide-f** [comment]
Hello @ekatef :)
As agreed, this PR is ready to squash and merge as the local check to verify that the google drive zip file is successfully automatically downloaded.
As that is done, the PR can be squashed and merged.

Congrats @Emre-Yorat89 and @ekatef :D

**Comment by ekatef** [comment]
> Hello @ekatef :) As agreed, this PR is ready to squash and merge as the local check to verify that the google drive zip file is successfully automatically downloaded. As that is done, the PR can be squashed and merged.
> 
> Congrats @Emre-Yorat89 and @ekatef :D

Hello @davide-f! Thanks a lot for checking :D
I confirm that automated download works properly 🎉

**Comment by ekatef** [comment]
Merged 🙂 
Thanks a lot @Emre-Yorat89 and @davide-f!
@Emre-Yorat89 congratulations with the finalised PR. Great work! 🥇

**Comment by Emre-Yorat89** [comment]
Thank you @ekatef and @davide-f for your reviews and resolving the conflicts on the PR. I am happy it is finalized :)

**Pull request merged**
